docker/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+    local var="$1"
+    local fileVar="${var}_FILE"
+    local def="${2:-}"
+    if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
+        echo >&2 "error: both $var and $fileVar are set (but are exclusive)"
+        exit 1
+    fi
+    local val="$def"
+    if [ "${!var:-}" ]; then
+        val="${!var}"
+    elif [ "${!fileVar:-}" ]; then
+        val="$(< "${!fileVar}")"
+    fi
+    export "$var"="$val"
+    unset "$fileVar"
+}
+
+file_env 'IOTA_AUTH_USER'
+file_env 'IOTA_AUTH_PASSWORD'
+
+if [[  -z "$IOTA_AUTH_ENABLED" ]]; then
+ echo "***********************************************"
+ echo "WARNING: It is recommended to enable authentication for secure connection"
+ echo "***********************************************"
+else
+    if  [[ -z "$IOTA_AUTH_USER" ]] || [ -z "$IOTA_AUTH_PASSWORD" ]]; then
+        echo "***********************************************"
+        echo "WARNING: Default IoT Agent Auth credentials have not been overridden"
+        echo "The connection to the Context broker fails"
+        echo "***********************************************"
+    else
+        echo "***********************************************"
+        echo "INFO: IoT Agent Auth credentials have been overridden"
+        echo "***********************************************"
+    fi
+fi
+
+pm2-runtime start /bin/iotagent-json
```

Comment:
I understand this comes from:

```
ENTRYPOINT ["pm2-runtime", "bin/iotagent-json"]
```

Should it be (as shown now)

```
pm2-runtime start /bin/iotagent-json
```

or actually:

```
pm2-runtime /bin/iotagent-json
```
?

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/3a5d1c5f5d74cc9b34444041157ce0f03f3ef7f0

Reply:
I see now it is:

```
pm2-runtime opt/iotajson/bin/iotagent-json
```

I think is safer to use absolute pathname here. Could you change it to 

```
pm2-runtime /opt/iotajson/bin/iotagent-json
```

please?

Reply:
Actually, I have realized we have this in dockerfile:

```
WORKDIR /opt/iotajson
```

so it could be even better this way (closer to the original one before this PR):

```
pm2-runtime bin/iotagent-json
```

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/20e74f925fbc7728990f3b4f2008e5a2247129de